engine/apps/alerts/migrations/0032_remove_alertgroup_slack_message_state.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.20 on 2023-09-04 12:46
+
+import common.migrations.remove_field
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('alerts', '0031_auto_20230831_1445'),
+    ]
+
+    operations = [
+        common.migrations.remove_field.RemoveFieldState(
+            model_name='AlertGroup',
+            name='slack_message',
+        ),
+    ]
```

Comment:
Remove `AlertGroup.slack_message` so the only FK left is `SlackMessage.alert_group`